ohttp/src/lib.rs

```diff
@@ -314,6 +311,27 @@ impl ClientRequest {
         Ok(Self { hpke, header })
     }
 
+    /// Reads an encoded configuration and constructs a single use client sender.
+    /// See `KeyConfig::encode` for the structure details.
+    #[allow(clippy::similar_names)] // for `sk_s` and `pk_s`
```

Comment:
The comment on this lint suppression seems wrong (I can't see why this warning would be a problem here).

Reply:
Fixed.